mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -1640,6 +1703,10 @@ struct Conv2DRewritePattern : public OpRewritePattern<T> {
           targetGemmDim2Attr.push_back(b.getNamedAttr(
               "names", b.getArrayAttr({b.getStringAttr(gemmKPad_name)})));
         }
+        // output of forward, gemmK = no * ho * wo
+        outputOobCheckDims.insert(nameToDims["no"]);
+        outputOobCheckDims.insert(nameToDims["ho"]);
```

Comment:
@ltqin can we use `needExtraPad` and `gemmKExtra` to help us understand should `no/ho/wo` all be inserted, or just `no`?

Reply:
same above